acceptable/__main__.py

```diff
@@ -28,11 +35,36 @@
     SafeRepresenter.add_representer(newdict, SafeRepresenter.represent_dict)
 
 
+def tojson_filter(json_object, indent=4):
+    return json.dumps(json_object, indent=indent, sort_keys=True)
+
+
+TEMPLATES = Environment(
+    loader=ChoiceLoader([
+        FileSystemLoader('./'),
+        PackageLoader('acceptable', 'templates'),
+    ]),
+    autoescape=False,
+)
+TEMPLATES.filters['tojson'] = tojson_filter
+
+
+class ForceAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        if not namespace.update:
+            if namespace.metadata:
+                namespace.metadata.close()  # supresses resource warning
+            parser.error('--force can only be used with --update')
+        else:
+            namespace.force = True
+
+
 def main():
     try:
         cli_args = parse_args()
         sys.exit(cli_args.func(cli_args))
     except Exception as e:
+        raise
         sys.exit(str(e))
```

Comment:
The `raise` leaves the subsequent code unreachable ...

Reply:
gah, was debugging code, sorry will remove